CMakeLists.txt

```diff
@@ -210,6 +210,11 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/sdbus-c++-config.cmake
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/sdbus-c++
         COMPONENT dev)
 
+if(BUILD_LIBSYSTEMD OR pkgcfg_lib_Systemd_systemd MATCHES "\.a$")
```

Comment:
I could not find any documentation on variables named according to the `pkgcfg_lib_*` pattern. Could you point me to some? Or aren't these internal variables?

In the latter case, wouldn't it be better to use documented variables -- i.e., `Systemd_LINK_LIBRARIES`, for example, instead of the internal `pkgcfg_lib_Systemd_systemd`?

Reply:
Good call. I'm no CMake expert, so I found that variable by diffing the build trees of a static and dynamic build. That was the only difference. I'm still surprised I overlooked the documented variable though. It's a `LIST`, but I was still able to use `MATCHES` against it.